src/Pages/ItemList/Components/CategoriesFilter.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { useEffect } from "react";
+import { useState } from "react";
```

Comment:
아래와 같이 한 줄로 나타낼 수 있겠네요! 
```suggestion
import React, { useState, useEffect } from "react";
```

Reply:
넵! 수정하겠습니다!